### ISSUE TITLE
Cannot run start-navdataupgrade on MT because tenant states OperationalWithSyncPending, no apps on tenant

### ISSUE THREAD

**Issue by ERPgruppenMW** [MicrosoftDocs/dynamics365smb-devitpro-pb#1875]
[Enter feedback here]
I cannot run Start-NAVDataUpgrade $instance -FunctionExecutionMode Serial -ContinueOnError -Tenant $tenantID
(By the way, an error in guide on "Run the data upgrade on the tenant" Step 4 "Start-NavDataUpgrade -ServerInstance ServerInstanceName> -FunctionExecutionMode Serial -ContinueOnError  ". A -tenant parameter is missing...

We have a NAV 2016 tenant with no extensions/apps installed. 
After running sync-navtenant state is OperationalWithSyncPending. 
(We remember to do Get-NAVTenant $ins -Tenant $tenantID -ForceRefresh)
When running Get-NAVAppInfo $ins -Tenant $tenantID we get no list.

So we are stuck in multi tenant upgrade :-|

Further info:
The tenant comes form a NAV 2016 app db with minor modifications and new tables. So we did force sync

BR
Mads

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6dde0890-47c3-1cfb-761b-321fe39f9e21
* Version Independent ID: de7dad4e-83a0-4a02-8caa-bde1187396c4
* Content: [Upgrading the Database to in a Multitenant Deployment - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/upgrade/upgrading-the-data-multitenant)
* Content Source: [dev-itpro/upgrade/Upgrading-the-Data-Multitenant.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/main/dev-itpro/upgrade/Upgrading-the-Data-Multitenant.md)
* Service: **dynamics365-business-central**
* GitHub Login: @jswymer
* Microsoft Alias: **jswymer**

**Comment by jswymer** [comment]
Hello, I have a couple questions, sorry if they seem irrelevant, but I'm just trying to make sure I understand the scenario.

From what I understand, you uninstalled all extensions. If this is the case, then the Get-NAVAppInfo -Server -Tenant cmdlet will not return anything because there are no extensions on the tenant.

Did you publish new versions of any extensions, for example, any of the Microsoft extensions?

**Comment by ERPgruppenMW** [comment]
Hi,
thanks for getting back.
In the meantime I found ot that this was an error on my behalf. 
My application database for v. 14 was faulted. I needed to create a new application DB (because of collation of tenant DB). I did so in CSide dev environment. I did not import whole database structure so sync and upgrade would have only system tables as part of the db create process from CSide dev environment. Once I imported whole structure the upgrade process worked fine.

Still a bug with the missing -tenant parameter though. 

Ok to close ticket/bug

BR Mads
